### PR TITLE
PHP7 Compatibility

### PR DESCRIPTION
**Pull request by sylus**
Based on: https://github.com/perftools/xhgui/pull/174

With tideways install added to .travis.yml

Two additional fixes to:

external/header.php
src/Xhgui/Saver.php

### CODE REVIEW

**Comment by sylus** [comment]
The coverage tests are now running but still 1 error detected.

**Comment by markstory** [comment]
Looking great!

File: external/header.php [hunk 1]
```diff
@@ -73,7 +73,7 @@
 }
 unset($dir);
 
-if (!extension_loaded('mongo') && Xhgui_Config::read('save.handler') === 'mongodb') {
+if ((!extension_loaded('mongo') || !extension_loaded('mongodb')) && Xhgui_Config::read('save.handler') === 'mongodb') {
```
Comment (krakjoe) [inline external/header.php]: This:

    !extension_loaded('mongo') || !extension_loaded('mongodb')

Should be:

    !extension_loaded('mongo') && !extension_loaded('mongodb')

**Comment by sylus** [comment]
I made the second correction, all we need to do is figure out the one failing test. Any ideas?

**Comment by markstory** [comment]
@sylus Thanks a bunch for this!. I fixed up the last test and have merged the changes in.

**Comment by sylus** [comment]
That is awesome, much appreciated! :)

**Pull request merged**